tests/test_discrete_problem.py

```diff
@@ -69,7 +69,7 @@ def test_solve_discrete_problem_no_shocks_illustrative():
     got = _solve_discrete_problem_no_shocks(
```

Comment:
Make two tests out of this -- same below. Second check won't run if the first one fails.